content/docs/intro/concepts/stack.md

```diff
@@ -61,24 +67,48 @@ test                                      2 weeks ago              121
 To select a stack that is part of an organization, use the fully-qualified stack name, either `orgName/stackName` or `orgName/projectName/stackName`:
 
 ```bash
-$ pulumi stack select acmecorp/prod
+$ pulumi stack select mycompany/prod
 
 $ pulumi stack ls
 NAME                                      LAST UPDATE              RESOURCE COUNT
-acmecorp/prod*                            4 hours ago              97
-acmecorp/staging                          4 hours ago              97
+mycompany/prod*                            4 hours ago              97
+mycompany/staging                          4 hours ago              97
 dev                                       n/a                      n/a
 ```
 
 ## Deploy a project
 
-To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "config" >}}) for the active stack.
+To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "/docs/intro/concepts/config" >}}) for the active stack.
+
+{{% notes "info"%}}
+Your stack can distinguish between execution for `preview` versus for `update` by using [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
+{{% /notes %}}
 
-> **Note:** Your stack can distinguish between execution for preview versus for update by using [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
+## Stack tags
+
+Stacks have associated metadata in the form of tags, with each tag consisting of a name and value. A set of built-in tags are automatically assigned and updated each time a stack is updated (such as `pulumi:project`, `pulumi:runtime`, `pulumi:description`, `gitHub:owner`, `gitHub:repo`, `vcs:owner`, `vcs:repo`, and `vcs:kind`). To view a stack's tags, run [`pulumi stack tag ls`]({{< relref "/docs/reference/cli/pulumi_stack_tag_ls" >}}).
+
+{{% notes "info" %}}
+Stack tags are only supported when logged into the [Pulumi Service backend]({{< relref "/docs/intro/concepts/state" >}}).
+{{% /notes %}}
+
+Custom tags can be assigned to a stack by running [`pulumi stack tag set <name> <value>`]({{< relref "/docs/reference/cli/pulumi_stack_tag_set" >}}) and can be used to customize the grouping of stacks in the [Pulumi Console](https://app.pulumi.com). For example, if you have many projects with separate stacks for production, staging, and testing environments, it may be useful to group stacks by environment instead of by project. To do this, you could assign a custom tag named `environment` to each stack. For example, running `pulumi stack tag set environment production` assigns a custom `environment` tag with a value of `production` to the active stack. Once you've assigned an `environment` tag to each stack, you'll be able to group by `Tag: environment` in the Pulumi Console.
```

Comment:
It'd be nice to show how to write, read/list (tags and tagged stacks), and delete stack tags at the command line (via an example), and maybe also how they appear when filtered in the Console.

Reply:
Agreed, but I'll tackle this as a follow up quality improvement work item.